ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeers.java

```diff
@@ -197,8 +196,11 @@ private boolean registerDisconnect(
         disconnectCallbacks.forEach(callback -> callback.onDisconnect(peer));
         peer.handleDisconnect();
         abortPendingRequestsAssignedToDisconnectedPeers();
-        LOG.debug("Disconnected EthPeer {}", peer.getShortNodeId());
-        LOG.trace("Disconnected EthPeer {}", peer);
+        if (peer.getReputation().getScore() > USEFULL_PEER_SCORE_THRESHOLD) {
+          LOG.debug("Disonnected USEFULL peer {}", peer);
+        } else if (!LOG.isTraceEnabled()) {
```

Comment:
We are checking for isTraceEnabled but logging at the debug level, should we make it a trace?

Reply:
I missed the "!", is that on purpose? Do we log only if Trace is not enabled?